### DISCUSSION TITLE
2 questions about cdArt display

### DISCUSSION THREAD

**Discussion by whydavid2022** [TT-ReBORN/Georgia-ReBORN#106] [Q&A] [answered]
1. Multi-CD cdArt In root dir of the album seems can't change automatically.
I used to organize the multi-cd into one single folder like this. Each file is tagged well such as disk number. cd2.png can't be displayed when I played CD2's songs. I study your instruction carefully. I think it should work but it fails. How can I solve it?
```
d:\Musics\Bon Jovi\Greatest Hits - The Ultimate Collection\cd.png
d:\Musics\Bon Jovi\Greatest Hits - The Ultimate Collection\CD1. 15. Bon Jovi - What Do You Got.m4a
d:\Musics\Bon Jovi\Greatest Hits - The Ultimate Collection\cd2.png
d:\Musics\Bon Jovi\Greatest Hits - The Ultimate Collection\CD2. 08. Bon Jovi - Blaze Of Glory.m4a
d:\Musics\Bon Jovi\Greatest Hits - The Ultimate Collection\cover.jpg
```
2. embedded cdArt (just the cdart) can't be display.
for single EP, I used to embed cover and cd artwork into the music file.  foobar will display disc art placeholder instead of embedded disk artwork.  is it possible to make foobar display the embeded disk artwok as cdArt?
![屏幕截图 2023-03-15 110157](https://user-images.githubusercontent.com/107458527/225194524-cb89f631-dc25-4b53-b91a-a21cfd7cbb5a.png)

thanks for your great job again. best regards.

**Comment by TT-ReBORN** [comment, marked as answer]
For Multi-CD cdArt in root dir, you need to use `cd1.png`, `cd2.png`, `cd3.png` etc.
If your cdArt is named only `cd.png`, the theme will always use it. This can only work, if your music album folder 
has CD1 and CD2 sub-directories and each cd.png is in it.

Embedded discArt is currently not supported for the Details panel, it does work for the Library though...

-TT

**Reply by brunop00** [reply]
and if the cdart image file is in artwork directory?

**Reply by silverwings15** [reply]
Artwork\cd1.png

and so on

**Comment by whydavid2022** [comment]
Got it. It works like a charm. Looking forward to next exciting update.

**Comment by brunop00** [comment]
> Artwork\cd1.png
> 
> and so on

how to edit this? where is this configuration?
most of my images is in that directory

**Reply by silverwings15** [reply]
you have all your artwork in a common directory, instead of in each album folder? can you screenshot for me please

**Reply by brunop00** [reply]
no. each album folder has an artwork folder inside. i managed to make it spin, but only one album. i will show the folder structure.
even organized that way, it doesn't work. it only work if the image format is .png? can you help me?

![1 who](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/29774927/ad9c1137-5a80-47c8-85db-8e8e0ba6f35d)
![2 who](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/29774927/24cee2cb-ac96-44f5-84e2-e7f20e74a65f)
![3 who](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/29774927/8f7ff166-2d2b-4d43-a465-0c6d99a89bb9)

**Reply by silverwings15** [reply]
![image](https://github.com/TT-ReBORN/Georgia-ReBORN/assets/10742312/a7417707-bd68-4374-a24f-41535f917413)

given your structure, you'd have to put front.* in the Front cover tab, artwork\cd.* in the Disc tab, artwork\back.* in the Back cover tab, and so on

**Reply by brunop00** [reply]
i think i found the problem, it only accept .png files. my image files are .jpeg. is there any component that converts several images together?

**Reply by TT-ReBORN** [reply]
JPG disc art is not supported, only PNG files. It would look stupid with the rotating white background color.
You should download transparent PNG disc art files from https://fanart.tv

When you are playing a song and you are in the Details panel, you can also use 
the shortcut -> right click on the album art > Get disc art

Or you can manually use Photoshop or GIMP and cut out the disc from the background color and save it as a PNG file.

**Reply by brunop00** [reply]
thanks for the instructions
